tools/scanner/scan.mk

```diff
@@ -58,7 +59,7 @@ scanner/scan:
 		-Dsonar.pullrequest.base=${PR_BASE}
 
 .PHONY: docker/scanner/build
-docker/scanner/build: scanner/image
+docker/scanner/build: output/docker_mlperf_scanner.stamp
```

Comment:
I thought the whole reason to have the `scanner/image` target is to avoid using the `.stamp` target. At the very least it's easier to read.
Am I missing something here?

Reply:
There is already a build image step before, so I though adding that `.stamp` would prevent the image from being built again if already happened, but somehow it didn't work. So I removed the `.stamp` dependency now.